tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -406,7 +407,7 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'responsive-embeds', $theme_supports );
 		$this->assertArrayHasKey( 'title-tag', $theme_supports );
 		$this->assertArrayHasKey( 'wp-block-styles', $theme_supports );
-		$this->assertCount( 21, $theme_supports );
+		$this->assertCount( 22, $theme_supports );
```

Comment:
This was bumped to `22` by 72380e31ca0bc05dca5617357412ab8fd346a583 (https://github.com/WordPress/wordpress-develop/pull/3205):

https://github.com/WordPress/wordpress-develop/blob/0b2f05a9f61c4032d0ef3138973b10adee8a032f/tests/phpunit/tests/rest-api/rest-themes-controller.php#L410

So we'll need to rebase and increment it again:

```suggestion
		$this->assertCount( 23, $theme_supports );
```